bin/alpen-bridge/src/rpc_server.rs

```diff
@@ -227,10 +228,15 @@ impl BridgeRpc {
                         (record, config)
                     })
                     .collect();
-                info!(cache_len=%cache_lock.len(), "RPC server Contracts cache initialized");
+                let after_num_contracts = refreshed_contracts.len();
 
+                let mut cache_lock = cached_contracts.write().await;
+                *cache_lock = refreshed_contracts;
                 // drop the lock!
```

Comment:
don't leave redundant comments lol

Reply:
redundant!? You don't know how many times I've got burned by a lock? At Alpen I think I've seen at least 2 times. I'm gonna die in this hill, this is not redundant. I'll remove the comment once we have STM-ish lock-free data structures in the Bridge 😄

Reply:
no what I mean is that the comment provides no additional value over just dropping the lock